src/main/java/com/github/princesslana/slothbot/Moderator.java

```diff
@@ -0,0 +1,88 @@
+package com.github.princesslana.slothbot;
+
+import com.eclipsesource.json.Json;
+import com.eclipsesource.json.JsonArray;
+import com.eclipsesource.json.WriterConfig;
+import com.google.common.base.Charsets;
+import com.google.common.io.MoreFiles;
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.Map;
+import java.util.Set;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class Moderator {
```

Comment:
I think the moderatorForRole suffix on everything in redundant, since it's implied by the purpose of the class anyway

Reply:
I'd even remove the Moderator. I think add/remove/contains etc are clear enough on their own.

Usage is like `moderator.addModerator`, don't need moderator twice.